src/seedu/addressbook/AddressBook.java

```diff
@@ -128,6 +129,10 @@
     private static final String COMMAND_HELP_WORD = "help";
     private static final String COMMAND_HELP_DESC = "Shows program usage instructions.";
     private static final String COMMAND_HELP_EXAMPLE = COMMAND_HELP_WORD;
+    
```

Comment:
Seems that you have changes from different activities mixed up :-).

Start a fresh branch from master for each of the activities to make the changes clean cut and manageable.